src/lib/controller.ts

```diff
@@ -26,10 +24,12 @@ export class AngusController {
         await this.ctx.stub.putState(key, data);
     }
 
-    async getModel(key:string) {
-        let ledgerKey:string = this.createCompositeKey(this.name, AngusModel.splitKey(key));
+    async getModel(key: string) {
+        let ledgerKey: string = this.createCompositeKey(this.name, AngusModel.splitKey(key));
         let data = await this.ctx.stub.getState(ledgerKey);
         let state: any;
+        this.ctx.getLogger('getModel').debug(`ledgerKey: ${ledgerKey}`);
+        this.ctx.getLogger('getModel').debug(`data: ${JSON.stringify(data, null, 2)}`);
```

Comment:
The new logs.
These would have helped me to resolve the getModel bug.

The rest of the changes are style normalization.